Core/StartupArgManager.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace LocalAdmin.V2.Core
+{
+    public sealed class StartupArgManager
+    {
+        /// <summary>
+        /// Path to startup arguments file.
+        /// </summary>
+        private const string StartupArgsPath = "laargs.txt";
+
+        /// <summary>
+        /// Merges Command-line arguments and arguments in <paramref name="StartupArgsPath"/>
+        /// </summary>
+        /// <param name="CMDArgs">Runtime Command-Line Arguments</param>
+        /// <returns>Merged Arguments</returns>
+        public static string[] MergeStartupArgs(string[] CMDArgs)
```

Comment:
If an exception happens here (and it can happen) we should just return the original array and print the exception.

And this can take `IEnumerable<string>` instead of `string[]` as an arg.

Reply:
Done, it's not fool proof, but as long as it works, that's what matters.